web/src/pages/team/wdobert.md

```diff
@@ -164,7 +164,7 @@ _Figure 1: Test pattern applied to our LED prototyping hardware by the latest dr
 
 #### Formalizing Main Application Architecture
 
-While developing the LED driver, I was simultaneously organzing the structure of our firmware application. Although I made important steps toward this goal last week, there was still some smoothing of rough edges to do. The entrypoint for our main application is now in *app_core.c*, and my intention is that any additional files that support our application in a central manner be named with *app_* prefix. A primary example of this is the debugging module I created (*app_debug.c*). At the moment, this module contains a function to print timestamped log messages to the integrated serial interface, but it will be expanded in the future to contain firmware debugging functions as needed. Obtaining an accurate timestamp for log messages necessitated enabling a hardware timer and writing a basic interrupt handler for it. As with *driver_led.c*, my goal is for all driver source files to be prefixed with *driver_* as they are created for new hardware (such as small panel displays).
+While developing the LED driver, I was simultaneously organzing the structure of our firmware application. Although I made important steps toward this goal last week, there was still some smoothing of rough edges to do. The entrypoint for our main application is now in _app_core.c_, and my intention is that any additional files that support our application in a central manner be named with \_app\__ prefix. A primary example of this is the debugging module I created (\_app_debug.c_). At the moment, this module contains a function to print timestamped log messages to the integrated serial interface, but it will be expanded in the future to contain firmware debugging functions as needed. Obtaining an accurate timestamp for log messages necessitated enabling a hardware timer and writing a basic interrupt handler for it. As with _driver_led.c_, my goal is for all driver source files to be prefixed with \_driver\_\_ as they are created for new hardware (such as small panel displays).
```

Comment:
A couple formatting errors:

![image](https://github.com/alanssitis/smart-air-hockey-table/assets/89476587/ba064894-1962-47d9-b3f5-aa53231c4955)
__app_ vs. \_app\_\_
_app_debug.c_ vs. \_app\_debug.c\_

![image](https://github.com/alanssitis/smart-air-hockey-table/assets/89476587/c30cdf50-0ba8-4628-a3f1-132ef8d26ab7)
__driver_ vs. \_driver\_\_

Reply:
Bro change _this__ to `this` please

Reply:
@WDobert follow @benjamin-owen's thing